### DISCUSSION TITLE
Welcome to E-TKT Discussions!

### DISCUSSION THREAD

**Discussion by andreisperid** [andreisperid/E-TKT#5] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by skrutt** [comment]
Hi again Andrei! Greetings from hackday.io.
Half a year later, i have finally got started, printed all the pieces and when I i got started on the electronics i realized that you did quite some changes.

Either way, i will press on with the older design, since i have all the parts.

So far so good, i have got a first flash of the firmware and connected to wifi.

But I do have a question, the documentation mentions to not forget about spiffs, i think I can get it going but have never used spiffs before. It seems to be missing since the page does not load.

I'm guessing i need the mkspiffs tool?

**Reply by skrutt** [reply]
Ok, found a how-to, now it seems to work :) https://randomnerdtutorials.com/esp32-vs-code-platformio-spiffs/

**Reply by andreisperid** [reply]
Hello @skrutt ! 

I swear I was just about to send the below message while I received yours:

Happy to know you are making an E-TKT for you :)

I don't know if you downloaded all the files back then, but since I've started designing the PCB there were a lot of changes. You might want to make sure all files are from the same context, or adopt the new one.

SPIFFS is simply the filesystem the ESP32 will use to serve the web app files. You need to send the files to the internal flash memory. You don't need to use any standalone tool for that, there are easier ways to do that:
- I recommend using Platformio: https://randomnerdtutorials.com/esp32-vs-code-platformio-spiffs/ 
- In case you are using the Arduino IDE: https://randomnerdtutorials.com/install-esp32-filesystem-uploader-arduino-ide/

I'd love to see your build, if you want, please create a dedicated post about your build :)

Cheers!
